pkg/authenticator/requests_test.go

```diff
@@ -0,0 +1,28 @@
+package authenticator
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestRequests(t *testing.T) {
+	Convey("LoginRequest", t, func() {
+		conjurVersion := "5"
+		authnURL := "dummyURL"
+		csrBytes := []byte("dummyCSRBytes")
+
+		Convey("Given a host's username prefix", func() {
+			usernamePrefix := "host.path.to.policy"
```

Comment:
Do you see any value to test also an empty Prefix ?

Reply:
we added input validation on the username that it includes a `host/` prefix so we don't have an empty prefix scenario